src/accessibility.md.njk

```diff
@@ -6,31 +6,79 @@ layout: layout-single-page-prose.njk
 
-You can see the [issues DAC raised and what the team has fixed](https://github.com/issues?q=is%3Aissue+archived%3Afalse+org%3Aalphagov+label%3A%22audit%3A+may-2019%22+).
+## Technical information about this website’s accessibility
+
+The Government Digital Service is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+
+This website is fully compliant with the Web Content Accessibility Guidelines version 2.1 AA standard.
+
+## How this website has been tested for accessibility
+
+This website was last tested on 7 October. The test was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
+
+DAC tested a sample of pages to cover the different content types in the GOV.UK Design System website. They were:
+
+- [the homepage](https://design-system.service.gov.uk/)
+- [a short content page](https://design-system.service.gov.uk/community/design-system-working-group)
+- [a long content page](https://design-system.service.gov.uk/styles/layout/)
+- [an overview page](https://design-system.service.gov.uk/community/)
+- [a styles page](https://design-system.service.gov.uk/styles/typography/)
+- [a component page](https://design-system.service.gov.uk/components/radios/)
+- [a pattern page](https://design-system.service.gov.uk/patterns/question-pages/)
+
+They also tested the global search functionality which appears in the header of every page in the GOV.UK Design System. 
+
+## How the GOV.UK Design System team makes this website accessible 
+
+The GOV.UK Design System team works hard to ensure that this Design System and GOV.UK Frontend, the codebase it uses, are accessible.
 
-There are a small number of [accessibility issues that the team still needs to fix](https://github.com/issues?q=is%3Aissue+archived%3Afalse+org%3Aalphagov+label%3A%22audit%3A+may-2019%22+is%3Aopen).
+Where possible, the team aims to research components and patterns with a representative range of users, including those with disabilities.
 
-Where possible, the team aims to research components and patterns with a representative range of users, including those with disabilities. 
+It also tests components to ensure they work with a broad range of browsers, devices and assistive technologies - including screen magnifiers, screen readers and speech recognition tools.
 
-It also tests components to ensure they work with a broad range of [browsers, devices and assistive technologies](https://github.com/alphagov/govuk-frontend#assistive-technology-support).
+This statement was prepared on 23 October 2019. It was last updated on 23 October 2019.
```

Comment:
There's a bit of a structural issue here. It's not clear this line refers to the statement as a whole, because there are h2s after '## Accessibility statement for the GOV.UK Design System website'. Would it work if all the headings after the initial h2 were h3s?

Reply:
In the sample statement it is at the end, which is something we could consider.

https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website

Reply:
Yeah it's a good point. I have used the heading levels as suggested in the [Sample accessibility statement](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website). I guess the options are:

- break that structure and make the H3s within the statement h3s.

or

- Move everything within the statement a level down, making the h2s h3s and the h3s h4s.

Do you have a preference?

Reply:
I had this in mind, if it makes sense (I think it's option 1?). It breaks the template structure but I think that's unavoidable because the template assumes the first heading is a h1 rather than a h2.

\#\# Accessibility statement for the GOV.UK Design System website
\#\#\# Reporting accessibility problems with this website
\#\#\# Enforcement procedure
\#\#\# Technical information about this website’s accessibility
\#\#\# How this website has been tested for accessibility
\#\# How the GOV.UK Design System team makes this website accessible
\#\# Using the GOV.UK Design System and GOV.UK Frontend in your service
\#\# If you are not using the GOV.UK Design System and GOV.UK Frontend

Reply:
Gotcha - sounds sensible to me. I'll change now.